style.css

```diff
@@ -164,9 +164,110 @@ ul li {
   transition: all 0.5s;
 }
 
+/* <------------------ Pop-up ---------------> */
+
+.back-pop {
+  position: absolute;
+  background-color: rgba(130, 134, 141, 0.841);
+  backdrop-filter: blur(5px);
+  width: 100%;
+  height: 2000px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.popup {
+  position: relative;
+  z-index: 200;
+  width: 70%;
+  margin-top: 50px;
+  background: var(--card-color);
+  border: 1px solid #cfd8dc;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 16px;
+}
```

Comment:
Please also fix the mobile popup as currently the close button is mixed with the texts and is not appearing as it is in the original design your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/202549529-cc86f14e-5e5d-4e7d-8f82-64020cfe6dc8.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/202549634-59a1334f-60cf-45d3-a9f6-08babf7bccf5.png)